src/databricks/labs/ucx/source_code/astlinter.py

```diff
@@ -0,0 +1,58 @@
+import ast
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MatchingVisitor(ast.NodeVisitor):
+
+    def __init__(self, node_type: type, match_nodes: list[tuple[str, type]]):
+        self.matched_nodes: list[ast.AST] = []
+        self._node_type = node_type
+        self._match_nodes = match_nodes
+
+    # pylint: disable=invalid-name
```

Comment:
```suggestion
```
can we fix the error and not silence it?

Reply:
Optionally, here as well:

https://github.com/databrickslabs/ucx/commit/c8c60379deda1a993e7df1f891fd50184759b476

Reply:
So, what's the invalid name here?

Reply:
it's `visit_Class`. Interestingly, Python core engine does not enforce pep